rmw/include/rmw/types.h

```diff
@@ -179,6 +179,8 @@ typedef struct RMW_PUBLIC_TYPE rmw_subscriptions_t
   size_t subscriber_count;
   /// Pointer to an array of void * pointers of subscriptions.
   void ** subscribers;
+  /// Pointer to an array rcutil timestamps, to be filled in same order as the above
+  rcutils_time_point_value_t *timestamps;
```

Comment:
Our style checking linter will choke on this otherwise. Here and below.

```suggestion
  rcutils_time_point_value_t * timestamps;
```